xonsh/aliases.py

```diff
@@ -55,6 +56,20 @@ def source_bash(args, stdin=None):
         env[k] = v
     return
 
+def bash_aliases():
+    try:
+        s = subprocess.check_output(['bash', '-i'], input='alias',
+                                    stderr=subprocess.PIPE,
+                                    universal_newlines=True)
+    except subprocess.CalledProcessError:
+        s = ''
+    items = [line.split('=', 1) for line in s.splitlines() if '=' in line]
+    aliases = [(item[0][6:],shlex.split(item[1].strip('\''))) for item in items]
+    return dict(aliases)
+
+
+BASH_ALIASES = bash_aliases()
```

Comment:
hmm, I think it would be cleaner to not have this here and instead import the `bash_aliases()` function in built_ins.